src/Microsoft.Azure.WebJobs.Host/InvocationFilterAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// This is the definition for the function filter attribute
+    /// </summary>
+    [CLSCompliant(false)]
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, Inherited = true, AllowMultiple = false)]
```

Comment:
I don't think this AttributeUsage on the base abstract class has any effect? If not, we can remove it. I think its the usage on subclasses that is honored by the reflection apis